### ISSUE TITLE
Copy/move axioms to *new* ontology gives error (in P5)

### ISSUE THREAD

**Issue by rsgoncalves** [protegeproject/protege#125]
When copying or moving axioms to a New ontology, after the very final step, the exception pasted below is thrown, and no action is done (no new ontology is created with the appropriate selection of axioms).

In the SelectTargetOntologyTypePanel class the targetOntologyID is set to null when the merge into new ontology button is selected. So in the MoveAxiomsWizard class we need to ensure a non-null target (in lines 212 and 247).

```
Error 1 Logged at Wed Feb 18 14:04:54 PST 2015
NullPointerException: null
    uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.contains(OWLOntologyManagerImpl.java:240)
    org.protege.editor.owl.ui.ontology.wizard.move.MoveAxiomsWizard.applyChanges(MoveAxiomsWizard.java:212)
    org.protege.editor.owl.ui.action.MoveAxiomsToOntologyAction.actionPerformed(MoveAxiomsToOntologyAction.java:23)
    javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
    javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2346)
    javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
    javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
    javax.swing.AbstractButton.doClick(AbstractButton.java:376)
    com.apple.laf.ScreenMenuItem.actionPerformed(ScreenMenuItem.java:125)
    java.awt.MenuItem.processActionEvent(MenuItem.java:669)
    java.awt.MenuItem.processEvent(MenuItem.java:628)
    java.awt.MenuComponent.dispatchEventImpl(MenuComponent.java:351)
    java.awt.MenuComponent.dispatchEvent(MenuComponent.java:339)
    java.awt.EventQueue.dispatchEventImpl(EventQueue.java:749)
    java.awt.EventQueue.access$400(EventQueue.java:97)
    java.awt.EventQueue$3.run(EventQueue.java:697)
    java.awt.EventQueue$3.run(EventQueue.java:691)
    java.security.AccessController.doPrivileged(Native Method)
    java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
    java.awt.EventQueue$4.run(EventQueue.java:719)
    java.awt.EventQueue$4.run(EventQueue.java:717)
    java.security.AccessController.doPrivileged(Native Method)
    java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    java.awt.EventQueue.dispatchEvent(EventQueue.java:716)
    java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by tudorache** [comment]
Confirmed: Refactor menu -> Copy, move, delete axioms -> any option -> copy to new ontology.

Workaround: Create a new ontology in the same workspace/window, select "Existing ontology", then select this new ontology, and save at the end.

Exception below
```
Error 1 Logged at Wed Feb 18 14:38:54 PST 2015
NullPointerException: null
    uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.contains(OWLOntologyManagerImpl.java:240)
    org.protege.editor.owl.ui.ontology.wizard.move.MoveAxiomsWizard.applyChanges(MoveAxiomsWizard.java:212)
    org.protege.editor.owl.ui.action.MoveAxiomsToOntologyAction.actionPerformed(MoveAxiomsToOntologyAction.java:23)
    javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
    javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2346)
    javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
    javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
    javax.swing.AbstractButton.doClick(AbstractButton.java:376)
    javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:833)
    javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:877)
    java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:289)
    java.awt.Component.processMouseEvent(Component.java:6525)
    javax.swing.JComponent.processMouseEvent(JComponent.java:3322)
    java.awt.Component.processEvent(Component.java:6290)
    java.awt.Container.processEvent(Container.java:2234)
    java.awt.Component.dispatchEventImpl(Component.java:4881)
    java.awt.Container.dispatchEventImpl(Container.java:2292)
    java.awt.Component.dispatchEvent(Component.java:4703)
    java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4898)
    java.awt.LightweightDispatcher.processMouseEvent(Container.java:4533)
    java.awt.LightweightDispatcher.dispatchEvent(Container.java:4462)
    java.awt.Container.dispatchEventImpl(Container.java:2278)
    java.awt.Window.dispatchEventImpl(Window.java:2739)
    java.awt.Component.dispatchEvent(Component.java:4703)
    java.awt.EventQueue.dispatchEventImpl(EventQueue.java:751)
    java.awt.EventQueue.access$500(EventQueue.java:97)
    java.awt.EventQueue$3.run(EventQueue.java:702)
    java.awt.EventQueue$3.run(EventQueue.java:696)
    java.security.AccessController.doPrivileged(Native Method)
    java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
    java.awt.EventQueue$4.run(EventQueue.java:724)
    java.awt.EventQueue$4.run(EventQueue.java:722)
    java.security.AccessController.doPrivileged(Native Method)
    java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    java.awt.EventQueue.dispatchEvent(EventQueue.java:721)
    java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
---------------------------------------------------------------------------------------------------

```

**Comment by rsgoncalves** [comment]
https://github.com/protegeproject/protege/commit/18756ebee17605aacbe6547591cdd2d92545264c fixed #125
